controllers/admin/AdminTranslationsController.php

```diff
@@ -271,11 +271,7 @@ protected function getModuleTranslations()
         if (!isset($_MODULE) && !isset($GLOBALS[$name_var])) {
             $GLOBALS[$name_var] = [];
         } elseif (isset($_MODULE)) {
-            if (is_array($GLOBALS[$name_var]) && is_array($_MODULE)) {
```

Comment:
This is a bit weird because the method so far did exactly what it stated in its description (for once we have a description on legacy code 😅) I'm a bit worried this could have unexpected side effects

Reply:
Yeah but its only used to fill translations in translation page. And it fills with default lang when it should leave it empty. So should i just close this pr and leave it like it is?